modules/tinymce/src/themes/silver/test/ts/browser/editor/bespoke/NamedStyleSelectFormatsTest.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  ApproxStructure, Assertions, Chain, Logger, Pipeline, Step, UiFinder
+} from '@ephox/agar';
+import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { TinyLoader } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+import Theme from 'tinymce/themes/silver/Theme';
+import * as MenuUtils from '../../../module/MenuUtils';
+
+interface StyleSelectMenuItem {
+  element: string;
+  title: string;
+}
+
+UnitTest.asynctest('browser.tinymce.themes.silver.bespoke.NamedStyleSelectFormatTest', (success, failure) => {
+  Theme();
+
+  TinyLoader.setupLight(
+    (editor, onSuccess, onFailure) => {
```

Comment:
Shouldn't this be on the line above, to be more consistent with the rest of the code (and line 16 of this file)

Reply:
It's the same indentation as other files in the same location the https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/themes/silver/test/ts/browser/editor/bespoke/SilverBespokeButtonsTest.ts#L14 for example. So maybe a cleanup task s is needed to fix all of them to the indentation you want. Now I'm keeping it consistent with the other tests.